src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -86,24 +96,16 @@ public void testDoGet() throws IOException, ServletException {
   public void testGetStats() throws IOException, ServletException {
     Mockito.when(mockRequest.getParameter("username")).thenReturn(null);
 
-    UUID fakeConversationId = UUID.randomUUID();
     List<Conversation> fakeConversationList = new ArrayList<>();
-    Conversation fakeConversation =
-        new Conversation(fakeConversationId, UUID.randomUUID(), "test_conversation", Instant.now());
     
     fakeConversationList.add(fakeConversation);
 
     Mockito.when(mockConvoStore.getAllConversations())
         .thenReturn(fakeConversationList);
 
     List<Message> fakeMessageList = new ArrayList<>();
-    fakeMessageList.add(
-        new Message(
-            UUID.randomUUID(),
-            fakeConversationId,
-            UUID.randomUUID(),
-            "test message",
-            Instant.now()));
```

Comment:
Similar question here, is there a reason why you don't need to necessarily make a new message?